packages/website/components/superhot.js

```diff
@@ -0,0 +1,258 @@
+    kwesforms.init()
+  }, [])
+
+  return (
+    <>
+      <div className="circle-container overflow-hidden">
+        <Circles />
+      </div>
+      <div className="bg-peach relative overflow-hidden text-center">
+        <div className="circle-container overflow-hidden -mt-[50px] lg:mt-[-70px]">
+          <Circles />
+        </div>
+        <div className="mx-auto max-w-7xl w-full pt-16 pb-20 lg:py-32 mt-12">
+          <div className="relative px-6">
+            <h1 className="font-open-sans text-xl lg:text-2xl leading-snug font-bold text-black ">
+              <span className="block">
+                Want more speed? Check out our premium features!
+              </span>
+            </h1>
+          </div>
+
+          <div className="px-6 max-w-6xl mx-auto mt-12">
+            <div className="relative  before:bg-[#F5C32C] before:absolute before:w-full before:h-full before:left-4 before:-top-4">
+              <div className="relative bg-white p-6 sm:p-10 lg:p-16 border-2 border-black">
+                <Img
+                  src="images/superhot-logo.svg"
+                  alt="logo"
+                  className="mb-12 mx-auto block"
+                />
+
+                <h4 className="text-lg md:text-xl leading-snug lg:text-2xl font-semibold max-w-4xl mx-auto">
+                  Perma-cache your NFT data on the edge to guarantee the fastest
+                  possible read performance by anyone on the web.
+                </h4>
+
+                <ul className="max-w-lg text-lg mx-auto mt-12">
+                  <li className="flex text-left my-8 items-start">
+                    <CheckMark />
+                    <p className="">
+                      50 ms time-to-first-byte for 95% of the world’s
+                      Internet-connected population
+                    </p>
+                  </li>
+                  <li className="flex text-left my-8 items-start">
+                    <CheckMark />
+                    <p className="">Best-in-class pricing</p>
+                  </li>
+                  <li className="flex text-left mb-8 mt-12 items-start">
+                    <CheckMark />
+                    <p className="">
+                      <small className="block absolute uppercase font-bold text-xs -mt-4 text-blue">
+                        Coming Soon
+                      </small>
+                      Centralized URL support (cache even non-IPFS data)
+                    </p>
+                  </li>
+                  <li className="flex text-left mb-8 mt-12 items-start">
+                    <CheckMark />
+                    <p className="">
+                      <small className="block absolute uppercase font-bold text-xs -mt-4 text-blue">
+                        Coming Soon
+                      </small>
+                      API for artists and collectors to perma-cache their
+                      collections.{' '}
+                      <a
+                        href=""
```

Comment:
We already have https://github.com/nftstorage/nftstorage.link/pull/71 so we can already put /api-docs in there. Also, should this open in new tab?